Src/Base/AMReX_Reduce.H

```diff
@@ -842,7 +842,7 @@ template <typename T, typename N, typename F,
 T Sum (N n, F&& f, T init_val = 0)
 {
     T r = init_val;
-#ifdef _OPENMP
+#ifdef _OPENMP // Note: intentionally not AMREX_USE_OMP
```

Comment:
The same argument for using `AMREX_USE_OMP` here as in `AMReX_NonLocalBCImpl.H`.

Reply:
```suggestion
#ifdef AMREX_USE_OMP
```